src/ng/services/dom/box.js

```diff
@@ -159,4 +159,8 @@ export default class Box extends BoxCore {
 
 		this.element.scrollTop = value;
 	}
+
+	get bounds() {
+		return this.element.getBoundingClientRect();
```

Comment:
the name should be "rect" at least and make it as function please or make scrollTop scrollLeft as getters